source/Properties/AssemblyInfo.cs

```diff
@@ -36,7 +36,7 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("2.2.0")]
-[assembly: AssemblyFileVersion("2.2.0")]
+[assembly: AssemblyVersion("2.0.1")]
+[assembly: AssemblyFileVersion("2.0.1")]

```

Comment:
We follow [SEMVER](http://semver.org/) so this will be 2.3.0 as it's a non-breaking feature enhancement